--- themes/tranquilpeak/layout/_partial/footer.ejs
+++ themes/tranquilpeak/layout/_partial/footer.ejs
@@ -2,0 +3,7 @@
+        <a href="https://github.com/MBRjun/MBRjun-Blog">GitHub 源代码</a> - <a href="https://github.com/MBRjun/MBRjun-Blog/blob/main/LICENSE">LICENSE(CC BY-SA 3)</a><br/>
+    </span>
+    <span class="text-small">
+        <a href="https://beian.miit.gov.cn" rel="nofollow">闽ICP备2025085162号-4</a> - 
+        <a href="http://www.beian.gov.cn" rel="nofollow"  >津公网安备12010102001086号</a><br/>
+    </span>
+    <span class="copyrights">
@@ -5 +12,2 @@
-</footer>
+</footer> 
+<script async defer data-website-id="8bccd081-57f0-4d61-908d-38e37d47241d" src="https://kbeacon.kmbr.dev/kbeacon.js?v=20241208-1" data-tag="MBRjun-Blog"></script>
