--- themes/tranquilpeak/layout/_partial/footer.ejs
+++ themes/tranquilpeak/layout/_partial/footer.ejs
@@ -2,0 +3,7 @@
+        <a href="https://github.com/MBRjun/MBRjun-Blog">GitHub 源代码</a> - <a href="https://github.com/MBRjun/MBRjun-Blog/blob/main/LICENSE">LICENSE(CC BY-SA 3)</a><br/>
+    </span>
+    <span class="text-small">
+        <a href="https://beian.miit.gov.cn" rel="nofollow">津ICP备20002670号</a> - 
+        <a href="http://www.beian.gov.cn" rel="nofollow"  >津公网安备12010102001086号</a><br/>
+    </span>
+    <span class="copyrights">
@@ -5 +12,2 @@
-</footer>
+</footer> 
+<script async defer data-website-id="d1e9e8b4-ed73-416a-b8e7-ae518e698da6" src="https://delightful.mbrjun.cn/delightful.js?v=2.2.0-1"></script>
