--- themes/tranquilpeak/layout/_partial/head.ejs
+++ themes/tranquilpeak/layout/_partial/head.ejs
@@ -185,6 +185,16 @@ var feeds = [];
     <!--SCRIPTS-->
 
     <!--SCRIPTS END-->
+    <!--PRELOAD-->
+    <meta http-equiv="x-dns-prefetch-control" content="on" />
+    <link rel="dns-prefetch" href="//github.githubassets.com">
+    <link rel="dns-prefetch" href="//avatars.githubusercontent.com">
+    <link rel="preconnect" href="https://cos.mbrjun.cn">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="/assets/fonts/GoogleSansDisplay-BR.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="/assets/fonts/GoogleSansDisplay-LR.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="/assets/fonts/fa-solid-900.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="/assets/fonts/fa-brands-400.woff2">
+    <!--PRELOAD END-->
 
     <%- partial('google-analytics') %>
     <%- partial('baidu-analytics') %>
